jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceFactory.java

```diff
@@ -146,19 +151,26 @@ static Resource combine(Resource... resources)
         return CombinedResource.combine(List.of(resources));
     }
 
+    static Resource resolveElseNew(Resource base, String pathOrUri, Function<String, Resource> factory)
```

Comment:
Javadoc?

Reply:
It was only used once. So I inlined it instead